bin/merge

```diff
@@ -90,3 +97,59 @@ else
     echo "error: ${repo}#${pull_request} has already been merged"
     exit 1
 fi
+
+}
+
+usage() {
+
+cat << EOF
+Usage: $0 [-h] [-r "REPO-NAME PR-NUMBER"]
```

Comment:
Since I'm using `getopts` to parse the flags and arguments, `<REPO-NAME> <PR-NUMBER>` need to come with a flag.

What about this?
`-r <REPO-NAME>,<PR-NUMBER>`